src/main/resources/db/changelog/changes/change-unique-index-in-user-tenant-table.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+  <changeSet id="MODCON-31@@update-user-tenant-index" author="singhadesh">
```

Comment:
One more thing, lets move this changeset to create-user-tenant table, to have all changesets for the corresponding table in one file , you can look create-tenant-table.xml as an example

Reply:
@SerhiiNosko :-  
Seems its not working when we drop index from same file.

POST request for mod-consortia-1.0.0-SNAPSHOT.34 /_/tenant failed with 400: Liquibase error: liquibase.exception.LiquibaseException: liquibase.exception.MigrationFailedException: Migration failed for changeset db/changelog/changes/create-user-tenant-table.xml::MODCON-31@@update-user-tenant-index::singhadesh:
Reason: liquibase.exception.DatabaseException: ERROR: index "username_tenant_id_idx" does not exist [Failed SQL: (0) DROP INDEX diku_mod_consortia.username_tenant_id_idx]

Reply:
it's bad practise to update already applied migrations, so `create-user-tenant` migration should be unchanged

Reply:
and we are not updating the same migration, we are introducing new change set that is located just in the same file to have all migrations set for particular table in the same place.
But if it not possible due to issue that Adesh mentioned - we can have separate files

Reply:
Fixed.